sys/contrib/ipfilter/netinet/mlfk_ipl.c

```diff
@@ -681,3 +693,12 @@ ipf_fbsd_sysctl_destroy(void)
 	return 0;
 }
 
```

Comment:
The whitespace at the end of this file is upstream, so I left it in place when adding the annotation.

Reply:
Didn't adding the annotations at the end cause way more merge conflicts? Shouldn't we put them after the copyright header instead?

Reply:
That's an orthogonal question I think.  For now the purecap branch follows current practice which is to stick them all at the end.